tests/middleware/test_security.py

```diff
@@ -94,11 +94,47 @@ def test_sts_no_include_subdomains(self):
         """
         With HSTS_SECONDS non-zero and HSTS_INCLUDE_SUBDOMAINS
         False, the middleware adds a "strict-transport-security" header without
-        the "includeSubDomains" tag to the response.
+        the "includeSubDomains" directive to the response.
         """
         response = self.process_response(secure=True)
         self.assertEqual(response["strict-transport-security"], "max-age=600")
 
+    @override_settings(
```

Comment:
use a single line here (we allow line length up to 119 chars when it improves readability)